### ISSUE TITLE
Disable the submit button if data is missing?

### ISSUE THREAD

**Issue by kwacky1** [HearthStats/HearthStats.net-Uploader#636]
Sometimes I get carried away with the game notes that I forget to check the missing data when the game mode and deck are not detected.
This results in an API error due to the missing data.
Would it be possible to disable the submit button if data is missing to prevent accidentally submitting incomplete data?

**Comment by tyrcho** [comment]
Yes it makes sense ! Maybe @Meowvw  you want to have a look ? It should be easy to implement.

**Comment by Meowvw** [comment]
Sorry, I didnt see the message here. I just checked, did you do that already?

**Comment by tyrcho** [comment]
Nope, it's not implemented yet, feel free to give it a try :)
